spotinst_sdk2/clients/elastigroup/__init__.py

```diff
@@ -1481,9 +1488,9 @@ def create_elastigroup(self, group):
 
         return ret_val
```

Comment:
Directly return from previous statement instead of creating a local variable. Make similar change throughout the class.

return formatted_response["response"]["items"][0]

Reply:
Done